.gitlab/lassen-jobs.yml

```diff
@@ -109,6 +109,11 @@ clang_9_0_0_memleak (build and test on lassen):
     ASAN_OPTIONS: "detect_leaks=1"
   extends: .build_and_test_on_lassen
 
+clang_9_0_0_datarace (build and test on lassen):
+  variables:
+    SPEC: "%clang@9.0.0 cxxflags=-fsanitize=thread"
```

Comment:
Adding the "+sanitizer_tests" caused it to fail the sanitizer tests during "make test" - is that ok?

Reply:
Yup, that option should maybe be renamed - it's only for the memory sanitizer tests, and requires the asan compiler flag.